docker-compose.yaml

```diff
@@ -165,8 +165,6 @@ services:
     logging: *default-logging
     container_name: airbyte-temporal
     restart: unless-stopped
-    ports:
-      - 7233:7233
```

Comment:
I'd suggest double checking that this also continues to work with the temporal UI that `docker-compose.debug.yaml` uses

Reply:
I will check that and add it to the debug before merging if needed

Reply:
@evantahler Tested locally, I didn't see any issues.